### PR TITLE
Add some compounds to vapor pressure batch7

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

**Review by qur786 — COMMENTED** [review]
All Good

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -1503,6 +1503,353 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C4: 2.0e-18,
+    C5: 6,
+    minimumTemperature: 300.03,
+    maximumTemperature: 766.8,
+    pressureAtMinimumTemperature: 7.09,
+    pressureAtMaximumTemperature: 3.097e6,
+  },
+
+  DiphenylAmine: {
+    name: 'Diphenyl amine',
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:1525]: name: 'Dipropyl amine'

**Review by saikat-samanta — APPROVED** [review]

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -1503,6 +1503,353 @@ export const DIPPRVaporPressureDictionary: Record<
     pressureAtMinimumTemperature: 4.47e-3,
     pressureAtMaximumTemperature: 3.199e6,
   },
+
+  //batch5
```
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:1507]:
```suggestion
```

Remove before merging

Comment (qur786) [inline src/data/vapor-pressure-constants.ts:1507]:
```suggestion

```

**Review by maneetgoyal — APPROVED** [review]
👍

**Review by qur786 — CHANGES_REQUESTED** [review]

**Review by qur786 — APPROVED** [review]

**Pull request merged**